### PR TITLE
updating repotoolset

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

**Comment by tmat** [comment]
>     <UsingToolPdbConverter>true</UsingToolPdbConverter>

This shouldn't be needed anymore.

---
Refers to: Directory.Build.props:16 in 64b0291. [](commit_id = 64b0291286b62608c099b83e4f7fd01f82634c94, deletion_comment = False)

**Comment by jmarolf** [comment]
>     <UsingToolPdbConverter>true</UsingToolPdbConverter>

Thanks, I'll remove it.

---
In reply to: [351870413](https://github.com/dotnet/roslyn-sdk/pull/27#issuecomment-351870413) [](ancestors = 351870413)

---
Refers to: Directory.Build.props:16 in 64b0291. [](commit_id = 64b0291286b62608c099b83e4f7fd01f82634c94, deletion_comment = False)

File: build/Toolset.proj [hunk 1]
```diff
@@ -4,6 +4,6 @@
     <TargetFramework>net462</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="RoslynTools.Microsoft.RepoToolset" Version="$(RoslynToolsMicrosoftRepoToolsetVersion)" />
+    <PackageReference Include="RoslynTools.RepoToolset" Version="$(RoslynToolsRepoToolsetVersion)" />
```
Comment (tmat) [inline build/Toolset.proj]: You can also delete nuget.config from the root directory and instead add 

<RestoreSources>[https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json](https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json)</RestoreSources>

here

Comment (tmat) [reply build/Toolset.proj]: The other feeds are included by RepoToolset automatically.

---
In reply to: [157092889](https://github.com/dotnet/roslyn-sdk/pull/27#discussion_r157092889) [](ancestors = 157092889)

Comment (jmarolf) [reply build/Toolset.proj]: xliff also gets pulled in?

**Comment by tmat** [comment]
>     <UsingToolPdbConverter>true</UsingToolPdbConverter>

Once this is checked in, we will also need to update the build definition to handle publishing symbols correctly.

---
In reply to: [351870580](https://github.com/dotnet/roslyn-sdk/pull/27#issuecomment-351870580) [](ancestors = 351870580,351870413)

---
Refers to: Directory.Build.props:16 in 64b0291. [](commit_id = 64b0291286b62608c099b83e4f7fd01f82634c94, deletion_comment = False)

File: build/Toolset.proj (hunk 1 above)
Comment (tmat) [reply build/Toolset.proj]: Yes.

---
In reply to: [157093125](https://github.com/dotnet/roslyn-sdk/pull/27#discussion_r157093125) [](ancestors = 157093125)

Comment (tmat) [reply build/Toolset.proj]: See [https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/DefaultVersions.props](https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/DefaultVersions.props)


---
In reply to: [157093293](https://github.com/dotnet/roslyn-sdk/pull/27#discussion_r157093293) [](ancestors = 157093293,157093125)

**Comment by jmarolf** [comment]
@tmat I'm getting two errors with this upgrade
`Unable to read content`: No  idea how to fix this.
```
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin/Templates/Templates.VisualStudio.2017/net461/Roslyn.SDK.Template.Wizard.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin/Templates/Templates.VisualStudio.2017/net461/Roslyn.SyntaxVisualizer.Control.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin/Templates/Templates.VisualStudio.2017/net461/Roslyn.SyntaxVisualizer.DgmlHelper.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin/Templates/Templates.VisualStudio.2017/net461/Roslyn.SyntaxVisualizer.Extension.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\cs\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\de\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\es\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\fr\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\it\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\ja\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\ko\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\pl\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\pt-BR\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\ru\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\tr\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\zh-Hans\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
  Unable to read content of C:\source\roslyn-sdk\artifacts\Debug\bin\Templates\Templates.VisualStudio.2017\net461\zh-Hant\Roslyn.SyntaxVisualizer.Extension.resources.dll: An item with the same key has already been added.
```
`Roslyn.SDK.VS2015.vsix has part.. which is not listed in the sign or external list` I think I know how to update the signdata to fix this.
```
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part Microsoft.CodeAnalysis.Analyzers.1.1.0.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part Microsoft.CodeAnalysis.Common.1.0.1.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part Microsoft.CodeAnalysis.CSharp.1.0.1.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part Microsoft.CodeAnalysis.CSharp.Workspaces.1.0.1.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part Microsoft.CodeAnalysis.VisualBasic.1.0.1.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part Microsoft.CodeAnalysis.VisualBasic.Workspaces.1.0.1.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part Microsoft.CodeAnalysis.Workspaces.Common.1.0.1.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part NuGet.CommandLine.2.8.5.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part System.Collections.Immutable.1.1.36.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part System.Composition.1.0.31.nupkg which is not listed in the sign or external list
  VSIX VSSetup/Roslyn.SDK.VS2015.vsix has part System.Reflection.Metadata.1.0.21.nupkg which is not listed in the sign or external list
```

**Comment by tmat** [comment]
Re the nuget one: you need an entry in SignToolData.proj:

https://github.com/dotnet/interactive-window/blob/master/build/SignToolData.json#L20-L26

**Comment by tmat** [comment]
👍

**Pull request merged**